src/main/app/controller/PostController.ts

```diff
@@ -31,6 +31,11 @@ export class PostController<T extends AnyObject> {
 
   private async saveAndSignOut(req: AppRequest<T>, res: Response, formData: Partial<Case>): Promise<void> {
     try {
+      for (const prop in formData) {
+        if (formData[prop] === Checkbox.Unchecked) {
+          formData[prop] = null;
```

Comment:
i assume it has to be null and '' breaks CCD?

Reply:
I've moved this to the checkbox parser to only run on save and sign out. `null` means user has not answered this field, while `""` means they've not selected it.